UM/Settings/ContainerStack.py

```diff
@@ -335,29 +336,35 @@ def deserialize(self, serialized):
         if "metadata" in parser:
             self._metadata = dict(parser["metadata"])
 
-        if "containers" in parser:
-            for index, container_id in parser.items("containers"):
-                containers = _containerRegistry.findContainers(id = container_id)
-                if containers:
-                    containers[0].propertyChanged.connect(self._collectPropertyChanges)
-                    self._containers.append(containers[0])
-                else:
-                    raise Exception("When trying to deserialize %s, we received an unknown ID (%s) for container" % (self._id, container_id))
-
-        elif parser.has_option("general", "containers"):
-            # Backward compatibility with 2.3.1: The containers used to be saved in a single comma-separated list.
-            container_string = parser["general"].get("containers", "")
-            Logger.log("d", "While deserializeing, we got the following container string: %s", container_string)
-            container_id_list = container_string.split(",")
-            for container_id in container_id_list:
-                if container_id != "":
+        if connect_signals:
```

Comment:
So if connect_signals is false, we do not add the containers and thus create an improperly deserialized container?